web/themes/custom/hoeringsportal/templates/components/promoted-dialogue.html.twig

```diff
@@ -0,0 +1,32 @@
+<div>
+  <div class="mb-3">{{ type }}: {{ label }}</div>
+  <div>
+    <div class="promoted-content p-4">
+      <h4 class="px-3">{{ left.leftLabel }}:</h4>
+      <div class="container">
+        <div class="row">
+          <div class="col-6">
```

Comment:
Might want to use `col-md-6` to wrap on mobile, and avoid very narrow column for the content.